README.md

```diff
@@ -194,6 +194,8 @@ day of week    0-7 (0 or 7 is Sunday, or use names)
 
 - `unrefTimeout`: [OPTIONAL] - Useful for controlling event loop behavior. More details [here](https://nodejs.org/api/timers.html#timers_timeout_unref).
 
+- `waitForCompletion`: [OPTIONAL] - If `true`, the job will wait for the `onTick` function to complete before stopping. Default is `false`.
```

Comment:
I'm not sure this is totally correct...the cron job itself (the thing keeping track of time) will not wait to stop if the callback is still executing, only any potential future callbacks, right? would a better be description be something like "If `true`, no additional instances of the `onTick` callback function will run unless the current `onTick` callback has completed."?

Reply:
```suggestion
- `waitForCompletion`: [OPTIONAL] - If `true`, no additional instances of the `onTick` callback function will run until the current onTick callback has completed. Any new scheduled executions that occur while the current callback is running will be skipped entirely. Default is `false`.
```

> "만약 `true`, 현재 콜백이 완료될 때까지 콜백 함수의 추가 인스턴스는 `onTick`실행되지 않습니다 `onTick`."와 같은 설명이 더 나을까요?

Yes, your newly proposed explanation concisely conveys that new `onTick` callbacks won't execute until the current callback completes. 
However, it might not clearly communicate that scheduled callbacks are skipped rather than waiting. That's why I've added that point in the new suggestion.